evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -583,11 +619,63 @@ contract FluxAggregator is AggregatorInterface, Owned {
    * Private
    */
 
+  function eligibleToSubmit(uint32 _round, bool supersedable)
```

Comment:
It feels odd to me to pass `supersedable` as an argument here. I think it would feel cleaner to give the `eligibleToSubmit` function just a single, `_roundId` param.